_zh-cn/overviews/scala3-book/types-type-classes.md

```diff
@@ -1,57 +0,0 @@
----
-title: Type Classes
-type: section
-description: This section introduces type classes in Scala 3.
-num: 60
-previous-page: ca-given-using-clauses
-next-page: ca-context-bounds
-
-partof: scala3-book
-overview-name: "Scala 3 — Book"
-layout: multipage-overview
-permalink: "/zh-cn/scala3/book/:title.html"
----
-
-A _type class_ is an abstract, parameterized type that lets you add new behavior to any closed data type without using sub-typing.
```

Comment:
Why did you delete this file? - it causes a link to break in `scala-for-python-devs.md`

Reply:
ok I see, its in English

Reply:
you will need to comment out links to this page in `scala-for-python-devs.md`